wavespectra/input/spotter.py

```diff
@@ -6,16 +6,206 @@
 
 """
 import glob
+import getpass
 import datetime
 import json
 import numpy as np
 import xarray as xr
+import pandas as pd
 from dateutil.parser import parse
 
 from wavespectra.specdataset import SpecDataset
 from wavespectra.core.attributes import attrs, set_spec_attributes
 
 
+def read_spotter_csv(filename):
```

Comment:
would be nice to handle (and concatenate) multiple `CSV` files as is currently done in the `JSON` Spotter workflow